src/classes/VOL_CTRL_SendBulkEmail.cls

```diff
@@ -118,28 +119,113 @@ public with sharing class VOL_CTRL_SendBulkEmail {
     }
     
     public string templateObject { get; set; }
-    
-    // action method to load up the email template the user has chosen
-    public PageReference LoadEmailTemplate() {
+
+    /*******************************************************************************************************************
+    * @description Sets the email body and subject from the template matching the template Id selected
+    * @return null
+    */
+    public PageReference loadEmailTemplate() {
+        isHtmlTemplate = false;
+        strSubject = null;
```

Comment:
How about just `subject` and `message` for these to avoid the use of Hungarian Notation.